scripts/hudson/narayana.sh

```diff
@@ -123,11 +123,38 @@ function check_if_pull_closed
 function build_narayana {
   echo "Building Narayana"
   cd $WORKSPACE
-  [ $NARAYANA_TESTS = 1 ] && NARAYANA_ARGS= || NARAYANA_ARGS="-DskipTests"
 
-  ./build.sh -Prelease,community,all$OBJECT_STORE_PROFILE -Didlj-enabled=true "$@" $NARAYANA_ARGS $IPV6_OPTS -B clean install
+  if [ -n "$JAVA_HOME_FOR_BUILD" ]; then
```

Comment:
I cannot find any documentation that describes what this `env` variable does. I can see it skipping some things but shouldn't the build instructions describe what it's for?

Reply:
That's a good point. I will add a comment in the `sh` script if it's fine.